components/profile_popover/profile_popover.tsx

```diff
@@ -400,45 +380,42 @@ class ProfilePopover extends React.PureComponent {
                 </div>,
             );
         }
-
         if (this.props.user.is_bot && !haveOverrideProp) {
             dataContent.push(
                 <div
                     key='bot-description'
                     className='overflow--ellipsis text-nowrap'
                 >
-                    {this.props.user.bot_description}
+                    {this.props.user.bot_description as ReactNode}
                 </div>,
             );
         }
-
         if (this.props.user.position && !haveOverrideProp) {
-            const position = this.props.user.position.substring(0, Constants.MAX_POSITION_LENGTH);
+            const position = (this.props.user?.position as string).substring(
```

Comment:
Instead of a potentially dangerous cast, we can coalesce to a blank string.
```suggestion
            const position = (this.props.user?.position || '').substring(
```

Reply:
Done